Makefile

```diff
@@ -0,0 +1,25 @@
+all: flake isort pipenv_check django_migrations_linter dotenv_linter vulture
```

Comment:
It's good practice to specify `SHELL` variable, cause you want to `bash` probably:

```
SHELL:=/usr/bin/env bash
```